tbx/project_styleguide/templates/patterns/organisms/footer/footer.html

```diff
@@ -42,13 +42,17 @@
 
     <div class="grid footer__bottom">
         <div class="grid__footer-logos">
-            {% with global_settings=settings.navigation.NavigationSettings %}
+            {% with settings.navigation.NavigationSettings as global_settings %}
```

Comment:
**Question** (completely opinionated and optional): Why revert to the old format? The new one permits setting multiple variables, and to me seems more familiar and easier to read as it mimics the appearance of variable assignment in code.

> The previous more verbose format is still supported: `{% with business.employees.count as total %}`  
> —https://docs.djangoproject.com/en/5.0/ref/templates/builtins/#with

The new syntax was introduced in Django 1.3 in 2011, so I can understand that we've not had much time to adjust: https://docs.djangoproject.com/en/5.0/releases/1.3/#improvements-to-built-in-template-tags

Reply:
Thanks @nimasmi

I changed to this while trying to make it work in the pattern library. For some reason, the pattern library component only rendered correctly when I changed to this syntax. I'll check again and see if there's a way I can use the new syntax without breaking the pattern library.

Reply:
don't spend too long worrying about that detail @engineervix - better to get this merged and focus on new stuff.

Reply:
Oh, curious. Thanks for explaining. Might this be something we need to feed back to the pattern library project?

Reply:
Once I check and confirm that I wasn't just using the wrong syntax in the YAML file,then I can raise it with the django-pattern-library project :+1: